packages/babel-preset-default/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Breaking Changes
 
+**Note** If you're using @wordpress/scripts for building JS scripts to target WordPress 6.5 or anterior, you should not upgrade to this version and continue using @wordpress/babel-preset-default@7.
```

Comment:
I had to look it up `anterior`. Maybe we should simplify the message here.

Reply:
Any suggestions? I guess it's easier to understand for me (coming from French)

Reply:
The phrase I usually see is "WP 6.5 or earlier". But let's ask native speakers 😅

Reply:
earlier is easier indeed. I spend let's say 20 seconds on it but didn't find the obvious word :P